### DISCUSSION TITLE
Forbidden (You don't have permission to access this resource.)

### DISCUSSION THREAD

**Discussion by margomius** [YOURLS/YOURLS#3198] [General]
Hi,

i updated from 1.7.9 to 1.8.2 and my URLs changed from `domain.com/short ` to `domain.com/yourls/short`

I didn't change anything apart from the update. I got a backup but I'd like to find the issue.

This is my directory structure:
domain.com
/yourls

This is my .htaccess in the main dir:
```
RewriteEngine On
RewriteBase /

# BEGIN YOURLS

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d

RewriteRule ^([0-9A-Za-z]+)/?$ https://domain.com/yourls/yourls-go.php?id=$1 [L]
RewriteRule ^([0-9A-Za-z]+)\+/?$ https://domain.com/yourls/yourls-infos.php?id=$1 [L]
RewriteRule ^([0-9A-Za-z]+)\+all/?$ https://domain.com/yourls/yourls-infos.php?id=$1&all=1 [L]

# END YOURLS
```

This is my yourls/.htaccess:
```
# BEGIN YOURLS
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /yourls/
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^.*$ /yourls/yourls-loader.php [L]
</IfModule>
# END YOURLS
```

In my config.php I have
`define( 'YOURLS_SITE', 'https://domain.com/yourls' );`

What am I doing wrong?

**Comment by ozh** [comment]
It's a server config outside of YOURLS that prevents from displaying directories without an index page.

**Reply by margomius** [reply]
Thanks. Any idea why it worked before I updated it?

**Reply by ozh** [reply]
What "worked" ? Seeing the root index page ? Well probably you had a root index page.

**Reply by margomius** [reply]
My URLs changed from /short to /yourls/short and I don't know why. I didn't change a index page or any htaccess settings. I just copied the new 1.8.2 files in my /yourls/ directory. How can I change it back? I want YOURLS to be in a subdirectory but still Short-URLS to be domain.com/short instead of domain.com/yourls/short.

**Reply by ozh** [reply]
Answer in the wiki
